src/helpers/generate.ratings.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable max-len */
```

Comment:
@nkpremices why are we disabling this?

Reply:
Oh!!! When I was working on this file, Eslint was complaining. I solved the problem but I forgot to remove this line. I'm going to do it